requirements.txt

```diff
@@ -1,98 +1,111 @@
-apipkg==1.5
+apipkg==2.1.0
 argcomplete==1.12.3
 asn1crypto==1.4.0
 atomicwrites==1.4.0
-attrs==20.3.0
-boto3==1.17.57
-botocore==1.20.57
-certifi==2020.12.5
-cffi==1.14.5
-cfn-flip==1.2.3
+attrs==21.2.0
+backports.zoneinfo==0.2.1
+boto3==1.19.0
+botocore==1.22.0
+Brotli==1.0.9
+certifi==2021.10.8
+cffi==1.15.0
+cfn-flip==1.3.0
 chardet==4.0.0
-Click==7.1.2
-coverage==5.5
-cryptography==3.4.7
-dateparser==1.0.0
+charset-normalizer==2.0.7
+click==7.1.2
+click-plugins==1.1.1
+cligj==0.7.2
+coverage==6.0.2
+# cryptography==3.4.7         # Version 35.0.0 breaks zappa=0.52.0 deployments. (yes, their versions went from 3.4.7, 3.4.8, then 35.0.0)
```

Comment:
Ha, someone probably fat-fingered a release number, and then it was too late to fix once it was out the door. Been there, done that.